gram.y

```diff
@@ -337,6 +338,39 @@ ifaceval	: T_MinRtrAdvInterval NUMBER ';'
 		{
 			iface->AdvRASolicitedUnicast = $2;
 		}
+		| T_AdvCaptivePortalAPI STRING ';'
+		{
+			const char *source = $2;
+			size_t len = strlen(source);
+
+			if (iface->AdvCaptivePortalAPI) {
+				flog(LOG_WARNING, "warning: AdvCaptivePortalAPI specified twice for interface %s in %s, line %d", iface->props.name, filename, num_lines);
```

Comment:
Line length not checked, but gut feeling says it is more than the 130 column width
as defined in https://github.com/radvd-project/radvd/blob/master/.clang-format

Reply:
Ack, I will split the flog() into two lines